src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/LaboratoryInvestigationNumericalResultTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Apart from `assertAll`, we seem to be preferring to use AssertJ over Jupiter assertions. It helps to keep the expected v actual ordering consistent (`testGetKey` has them the wrong way around for Jupiter).

`assertThatIllegalArgumentException().isThrownBy(ThrowingCallable).withMessage("exception message")`
`assertThatNullPointerException().isThrownBy(ThrowingCallable).withMessage("exception message")`
`assertThat(actual).isEqualTo(expected)`
`assertThatNoException().isThrownBy(ThrowingCallable)`